app/controllers/main.js

```diff
@@ -26,7 +26,7 @@ angular.module('coalaHtmlApp')
             "message":  el.message,
             "origin":   el.origin,
             "severity": el.severity
-          });
+          })});
```

Comment:
@Nosferatul  move the closing brackets + parenthesis pair of `forEach` to new line. This looks better :)
```javaScript
var parseResult = function(el){
  el.affected_code.forEach(function(sourceRange){
    if(){
    ..
    }
  });
})
```